source/main.cpp

```diff
@@ -197,6 +195,13 @@ int main(void) {
     currentScene->render(gRenderer);
 
     SDL_RenderPresent(gRenderer);
+
+    // Update controllers roughly every second
+    if(frame_counter % 60 == 0) {
+      update_controllers();
```

Comment:
I'm certified stupid, but wouldn't this attempt to open the same gamecontroller(s) once every second?

Reply:
Correct. This should allow controllers to be plugged in after the app is loaded. Looks to me to be pretty safe to call this often. https://github.com/XboxDev/nxdk-sdl/blob/a2a2d9ca1a5a7082c830490a0b110829939de3a7/src/joystick/SDL_gamecontroller.c#L1548